test/functional/interface_zmq.py

```diff
@@ -82,23 +88,46 @@ def run_test(self):
 
     # Restart node with the specified zmq notifications enabled, subscribe to
     # all of them and return the corresponding ZMQSubscriber objects.
-    def setup_zmq_test(self, services, recv_timeout=60, connect_nodes=False):
+    def setup_zmq_test(self, services, recv_timeout=60, sync_blocks=True):
```

Comment:
nit, can enforce named args with 
```suggestion
    def setup_zmq_test(self, services, *, recv_timeout=60, sync_blocks=True):
```
or alternatively for all args
```diff
@@ -88,7 +88,7 @@ class ZMQTest (BitcoinTestFramework):
 
     # Restart node with the specified zmq notifications enabled, subscribe to
     # all of them and return the corresponding ZMQSubscriber objects.
-    def setup_zmq_test(self, services, recv_timeout=60, sync_blocks=True):
+    def setup_zmq_test(self, *, services, recv_timeout=60, sync_blocks=True):
         subscribers = []
         for topic, address in services:
             socket = self.ctx.socket(zmq.SUB)
@@ -137,7 +137,7 @@ class ZMQTest (BitcoinTestFramework):
         self.restart_node(0, ["-zmqpubrawtx=foo", "-zmqpubhashtx=bar"])
 
         address = 'tcp://127.0.0.1:28332'
-        subs = self.setup_zmq_test([(topic, address) for topic in ["hashblock", "hashtx", "rawblock", "rawtx"]])
+        subs = self.setup_zmq_test(services=[(topic, address) for topic in ["hashblock", "hashtx", "rawblock", "rawtx"]])
 
         hashblock = subs[0]
         hashtx = subs[1]
@@ -212,7 +212,7 @@ class ZMQTest (BitcoinTestFramework):
 
         # Should only notify the tip if a reorg occurs
         hashblock, hashtx = self.setup_zmq_test(
-            [(topic, address) for topic in ["hashblock", "hashtx"]],
+            services=[(topic, address) for topic in ["hashblock", "hashtx"]],
             recv_timeout=2)  # 2 second timeout to check end of notifications
         self.disconnect_nodes(0, 1)
 
@@ -262,7 +262,7 @@ class ZMQTest (BitcoinTestFramework):
         <32-byte hash>A<8-byte LE uint> : Transactionhash added mempool
         """
         self.log.info("Testing 'sequence' publisher")
-        [seq] = self.setup_zmq_test([("sequence", "tcp://127.0.0.1:28333")])
+        [seq] = self.setup_zmq_test(services=[("sequence", "tcp://127.0.0.1:28333")])
         self.disconnect_nodes(0, 1)
 
         # Mempool sequence number starts at 1
@@ -414,7 +414,7 @@ class ZMQTest (BitcoinTestFramework):
             return
 
         self.log.info("Testing 'mempool sync' usage of sequence notifier")
-        [seq] = self.setup_zmq_test([("sequence", "tcp://127.0.0.1:28333")])
+        [seq] = self.setup_zmq_test(services=[("sequence", "tcp://127.0.0.1:28333")])
 
         # In-memory counter, should always start at 1
         next_mempool_seq = self.nodes[0].getrawmempool(mempool_sequence=True)["mempool_sequence"]
@@ -514,7 +514,7 @@ class ZMQTest (BitcoinTestFramework):
 
     def test_multiple_interfaces(self):
         # Set up two subscribers with different addresses
-        subscribers = self.setup_zmq_test([
+        subscribers = self.setup_zmq_test(services=[
             ("hashblock", "tcp://127.0.0.1:28334"),
             ("hashblock", "tcp://127.0.0.1:28335"),
         ], sync_blocks=False)
```

Reply:
Thanks, I decided for the first variant, i.e. enforcing named args for `recv_timeout` and `sync_blocks`.